zb-classifier.js

```diff
@@ -897,6 +897,63 @@ class ZigbeeClassifier {
     );
   }
 
+  addSeCurrentSummDeliveredProperty(node, seMeteringEndpoint) {
+    this.addProperty(
+      node,                             // device
+      '_counterMul',                    // name
+      {// property description
+        type: 'number',
+        readOnly: true,
+      },
+      PROFILE_ID.ZHA,                   // profileId
+      seMeteringEndpoint,               // endpoint
+      CLUSTER_ID.SEMETERING,            // clusterId
+      'currentSummDeliveredMultiplier', // attr
+      '',                               // setAttrFromValue
+      'parseNumericAttr',               // parseValueFromAttr
+      null,                             // configReport
+      1                                 // defaultValue
+    );
+    this.addProperty(
+      node,                           // device
+      '_counterDiv',                  // name
+      {// property description
+        type: 'number',
+        readOnly: true,
+      },
+      PROFILE_ID.ZHA,                 // profileId
+      seMeteringEndpoint,             // endpoint
+      CLUSTER_ID.SEMETERING,          // clusterId
+      'currentSummDeliveredDivisor',  // attr
+      '',                             // setAttrFromValue
+      'parseNumericAttr',             // parseValueFromAttr
+      null,                           // configReport
+      // use a default of 1000 for now until
+      // new attribute definitions are available
+      1000                            // defaultValue
+    );
+    this.addProperty(
+      node,                           // device
+      'counter',                      // name
+      {// property description
+        // even though the content is not instantaneous power, this is the only
+        // property type with watt as unit
+        '@type': 'InstantaneousPowerProperty',
```

Comment:
Please remove the comment and the `@type` field. It will work just fine without it.